guest/testdata/cyclestate/main.go

```diff
@@ -186,6 +189,10 @@ func (statePlugin) Reserve(state api.CycleState, pod proto.Pod, nodeName string)
 func (statePlugin) Unreserve(state api.CycleState, pod proto.Pod, nodeName string) {
 	mustFilterState(state)
 }
+func (statePlugin) Permit(state api.CycleState, pod proto.Pod, nodeName string) (status *api.Status, timeout time.Duration) {
+	// Actually, it is not called from the cycle test, but it needs for Permit plugin.
```

Comment:
Why is it not called? In other words, can we change the test and this `Permit` to make sure the cycle state is propagated to Permit correctly?

Reply:
Thanks,
I guess it makes sense to verify that the `preFilterStateKey` should be propagated on Permit since the existing _cycle state_ test implementation always calls prefilter, right?

Reply:
Yup, that'd be nice.